conan/tools/cmake/cmake.py

```diff
@@ -73,14 +73,14 @@ def __init__(self, conanfile, parallel=True):
 
         self._cmake_program = "cmake"  # Path to CMake should be handled by environment
 
-    def configure(self, source_folder=None):
+    def configure(self, build_script_folder=None):
         # TODO: environment?
         if not self._conanfile.should_configure:
             return
 
         source = self._conanfile.source_folder
-        if source_folder:
-            source = os.path.join(self._conanfile.source_folder, source_folder)
+        if build_script_folder:
+            source = os.path.join(self._conanfile.source_folder, build_script_folder)
```

Comment:
Probably I'd refactor this to name it ``cmakelist_folder``, to clearly differentiate it might not be the root src folder, and just in case somewhere else in the code used the ``source`` folder (but it is not an issue here, as it is not used), but it is ok, just an internal detail.